README.md

```diff
@@ -21,7 +21,13 @@ The following features however are yet to be implemented:
 * H264 Error Concealment (FEC and active error recovery however are implemented)
 * Touchpad support (Triggering the Touchpad Button is currently possible by pressing `T` on the keyboard)
 * Rumble
-* Configurable Keybindings
+* Configurable Keybindings. Though, for the moment you can use the following:
+  * `\` -> Square
+  * `]` -> Triangle
+  * `⏎` -> X
+  * `⌫` -> Circle
```

Comment:
Please use the "correct" Chiaki notation, i.e. Box, Pyramid, Cross, Moon 😉️

Reply:
Done. 

Out of curiosity, where did you pick those names from? Is there some sort of standard I don't know about?